src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine{
+    @Override
+    public void doWork() {
+        System.out.println("Excavator is work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator is stop work");

```

Comment:
Similarly, the message "Excavator is stop work" should be rephrased for clarity and grammatical correctness. A better message would be "Excavator stopped working".